diff --git a/cmake/config-ix.cmake b/cmake/config-ix.cmake
index 8892964..2768728 100644
--- a/cmake/config-ix.cmake
+++ b/cmake/config-ix.cmake
@@ -80,7 +80,5 @@ if(FUCHSIA)
 else()
   check_library_exists(dl dladdr "" LIBCXXABI_HAS_DL_LIB)
   check_library_exists(pthread pthread_once "" LIBCXXABI_HAS_PTHREAD_LIB)
-  check_library_exists(c __cxa_thread_atexit_impl ""
-    LIBCXXABI_HAS_CXA_THREAD_ATEXIT_IMPL)
   check_library_exists(System write "" LIBCXXABI_HAS_SYSTEM_LIB)
 endif()
